azkaban-webserver/src/package/conf/azkaban.properties

```diff
@@ -26,6 +26,7 @@ mysql.numconnections=100
 velocity.dev.mode=false
 
 # Azkaban Jetty server properties.
+jetty.hostname=localhost
```

Comment:
Could you update the comment to say that even on tomcat the jetty.* properties are used to generate links for email notifications?